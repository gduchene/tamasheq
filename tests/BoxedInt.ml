let x = Int32.of_int 1
let y = Int64.of_int 1
let z = Int64.to_int32 y |> Int32.add x |> Nativeint.of_int32

let a = Int32.to_int     x
let b = Int64.to_int     y
let c = Nativeint.to_int z
