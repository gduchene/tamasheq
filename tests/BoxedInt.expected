BoxedInt_0	=	[6] {
    (no name)	=	<abstr:255>
    (no name)	=	<abstr:255>
    (no name)	=	<abstr:255>
    (no name)	=	1
    (no name)	=	1
    (no name)	=	2
}
